<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="VLMainLayout">
      <item>
       <layout class="QHBoxLayout" name="HLMainLayout">
        <item>
         <layout class="QVBoxLayout" name="VLAVCameras">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Monocam:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="PBSearchMonocam">
              <property name="text">
               <string>Search Cameras</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PBFreerunMonocam">
              <property name="text">
               <string>Freerun Preview</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Detected Cameras:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="ListDetectedMonocam">
            <property name="maximumSize">
             <size>
              <width>475</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="HLCameraSettings">
            <item>
             <layout class="QVBoxLayout" name="VLCameraSetting">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Exposure:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LEExposureMonocam"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="VLCameraSetting_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Amplifier Gain [dB]:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LEAmpGainMonocam"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="VLCameraSetting_4">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Future Value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="LEAmpGainMonocam_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="PBWriteToCamera">
            <property name="text">
             <string>Write Settings to Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGraphicsView" name="GVPreviewMonocam">
              <property name="minimumSize">
               <size>
                <width>415</width>
                <height>309</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>415</width>
                <height>309</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="VLIRCameras">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>475</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Xi400 Thermal Camera:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="HLXiDetails">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Xi400 SN:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LEXiSerialNumber"/>
            </item>
            <item>
             <widget class="QPushButton" name="PBXiToggleConnect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Image Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxXiFormat"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Sensor Range:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxXiTempRange"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QGraphicsView" name="GVXiPreview">
              <property name="minimumSize">
               <size>
                <width>382</width>
                <height>288</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>382</width>
                <height>288</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="VLDAQmx">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>475</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>DAQmx:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="PBDAQSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PBDAQConnToggle">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="CameraSyncLayout">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Camera Sync Signal:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxCamSyncSignal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Laser On Signal:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxLaserOnSignal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Oxygen PPM Analog [0-10V]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxOxyPPMSignal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Moisture PPM Analog [0-10V]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxMoisturePPMSignal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Laser Power % Analog [0-10V]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxLaserPowerPctSignal"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>X Axis Encoder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxXEncoderA"/>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxXEncoderB"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Y Axis Encoder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxYEncoderA"/>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxYEncoderB"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Z Axis Encoder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxZEncoderA"/>
            </item>
            <item>
             <widget class="QComboBox" name="CBoxZEncoderB"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_27">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Live Values:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="4" column="1">
             <widget class="QLabel" name="LPowerPctState">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0.00V</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="LOxyPPMState">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0.00V</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>CameraSync</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="LXCount">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>00000</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="label_53">
              <property name="text">
               <string>YCount</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>OxyPPM</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="LCameraSyncState">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Off</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_37">
              <property name="text">
               <string>XCount</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>Power%</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="LPewPewOnState">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Off</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>MoisturePPM</string>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLabel" name="LWaterPPMState">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0.00V</string>
              </property>
             </widget>
            </item>
            <item row="5" column="5">
             <widget class="QLabel" name="LZCount">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>00000</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="LYCount">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>00000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>LaserOn</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <widget class="QLabel" name="label_51">
              <property name="text">
               <string>ZCount</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLExperimentDetails">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sync Microcontroller:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBoxMicroPort">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBMicroRefresh">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBMicroConnect">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sync Frequency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBoxSyncFreq"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Frames to Capture:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEFramesToGrab">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLOutputInfo">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Output Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEOutputDirectoryPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="PBBrowseOutputPath">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="PBStartExperiment">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Synchonized Capture</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
